src/analyze-trace-utilities.ts

```diff
@@ -8,6 +8,9 @@ import normalizePositions = require("./normalize-positions");
 import simplify = require("./simplify-type");
 import { EventSpan, ParseResult } from "./parse-trace-file";
 
+// @ts-ignore - no types
+import jsonstream = require("jsonstream-next");
```

Comment:
Let's wait on https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64292

Reply:
Totally possible to wait, but it's also just an additional usage of this existing library

Reply:
Hah - I totally missed that this was already used by our package.

Reply:
```
src/simplify-types-file.ts:63:19 - error TS2339: Property '_flush' does not exist on type 'ReadWriteStream'.

63         transform._flush = newFlush;
```

Reply:
Looks like the return type of `jsonstream.parse()` is the return of `@types/through2`

Reply:
How would you feel about adopting the types in a separate PR?

Reply:
I'd sooner see us write `(transform as any)._flush` (to make it abundantly clear we're doing something unsafe) for that code than not use the types at all.

Reply:
Before I merge, I'll need to try it out locally - but by that point, https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64383 will probably be merged anyway.